### DISCUSSION TITLE
[Weekly Meeting] [2024-03-15] - Meeting Notes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#1011] [Weekly Meetings]
- [ ] Quarkus 3.8.1 Upgrade
    - [ ] Optaplanner Operator Fix (*) - https://issues.apache.org/jira/browse/LEGAL-669
    - [ ] DevUI fixes
- [ ] Unblock Apache KIE 10.0.0 Release
    - [ ] Heated by healthy debate in ML, move us forward as Community as a safe environment to share and express perspectives and opinions
    - [ ] New info: some references in Apache code are referencing code that is not part of Apache (hosted in kiegroup).
    - [ ] Several lefover kiegroup references.
    - [ ] Dashbuilder had references to kie-soup (v7!)
    - [ ] knative cli is referencing a tag from kiegroup fork of kie-tools in kiegroup
    - [ ] Additional information will be added to Tiago's proposal to add the final revert procedures
    - [ ] Contention point was most localized on temporary meaning.. so incorporated into procedure aims to help
    - [ ] No need to fear that changes won't be reverted, as it has been documented and agreed. Nobody can't hold the revert because this is documented.
    - [ ] Amend the proposal with a section that describe exactly what is temp and will be reverted. With a conscious decision that this will break dev workflow and release. 
    - [ ] For Kris input is that mostly should be only documented to remove the folders images and operator from the kie-tools repo
    - [ ] Need additional discussion on some details of the proposal, like the impact of enable/disable CI from existing repo to avoid overlap build between original repos and copied code over kie-tools.
    - [ ] Gabriele: He would prefer to not undo the work, instead start sooner to discuss the longer term solution.
    - [ ] Not call a vote right now... maybe not even call for a vote. Tiago consider that we don't need a vote.
    - [ ] Francisco: even if the merge will take a while, and we save the biggest impact on CI. And we can have the release. We need have this discussion.
    - [ ] Francisco: leave devui in -apps. Why not? Circular dependency.
    - [ ] Release
       - [ ] Dominik: dry run release? what's the plan? Tiago. He drafted a step by step, dry run happened on Tiago's head :)
       - [ ] Tiago assumed that release for Cat A (runtimes), and the kie-tools will follow. Please add to the proposal who knows more about release.
       - [ ] Tibor. We also have staging for runtime. This could be considered as the buffer for the release assessment.
       - [ ] Tibor: It's required and assumed someone will drive the CI release. It won't be a button press to release this time. 
       - [ ] Alex: fair to say that we'll know more once we start. may work at first time, may need a few tries.
  - [ ] Discuss the future
    - [ ] Alex: Focus now on release, and after ready for release... we start the discussions on the shape of the future.
- [ ] Communication Channel
    - [ ] Gabriele: propose to have meeting instead ML to discuss the longer term solution. As detailed thread is hard to follow. As a task-force. Tibor suggestion to use Discussion. Tiago: fine to meet and discuss. Keep in mind that ML is the source of thruth (Dominik!). ML is more inclusive. 10hours + to write proposal was positive to evaluate all details. Alex. Calls are good to have conceptual level agreement, but hard to discuss on details. ML provides the opportunity to get in details. Kris: it's not call or ML, it can be both. ML is a difficult to `convince` and make yourself better understood. Tiago. Hard to agree on preferences calls vs ML. Apache has a definition of procedure, if we decide to make meetings... Tiago personally met with several stakeholders before the ML email. Not only after the email that he had the time to process all the knowledge shared to find also issues on the conversations.

-----
Recording

https://us06web.zoom.us/rec/share/VJMZSl-Pk148M90Z56jhRl9OsFTRcnra7RIX1uXy3QfeOP_bdmhNPJyhM87NGXzo.GGPB6KlCgQ2nx7QO
Passcode: =QL1H#YZ

**Comment by LuboTerifaj** [comment]
@porcelli the recording is only 12 seconds long. Is it a correct URL?

**Reply by porcelli** [reply]
ops.. sorry. just updated the correct one!
